cmd/harmony/main.go

```diff
@@ -427,6 +442,8 @@ func main() {
 	utils.SetLogContext(*port, *ip)
 	utils.SetLogVerbosity(log.Lvl(*verbosity))
 
+	fmt.Println(os.Args)
+	fmt.Println(*minPeers)
 	initSetup()
 	nodeConfig := createGlobalConfig()
```

Comment:
There are two many `if` conditions in this file,
 maybe we could simply have if `*isExplorer nodeConfig := createExplorerConfig() else nodeConfig := createGlobalConfig()` or we want to keep createGlobalConfig(), the if conditions in that function could be factored out into their own fucntions. e.g. `if *isExplorer nodeConfig := createExplorerConfig() else nodeConfig := createConsensusNodeConfig()` etc. Will make for better readability.